dist/boards/slstk3400a/doc.txt

```diff
@@ -22,19 +22,19 @@ actively measure the power consumption of your hardware and code, in real-time.
 | RAM             | 8.0KiB                                               |
 | Flash           | 64.0KiB                                              |
 | EEPROM          | no                                                   |
-| Frequency       | up to 24 MHz                                         |
+| Frequency       | up to 25 MHz                                         |
```

Comment:
You are right, but I will fix this later. The field is filled with the HFXO value, which is 24MHz for this board.

Reply:
Okay.

Reply:
You are right, I'll update this tonight.